src/views/News/components/NewsItem/newsItem.scss

```diff
@@ -13,6 +13,10 @@
     transition: background-color 150ms;
 }
 
+.text-link:hover {
```

Comment:
Good idea - always wise to lean on the obvious side when naming things. I'll probably use the dashes for consistency since other styles referenced in the NewsItem file uses the same convention.

Reply:
Or maybe a hybrid of both for naming conventions - like news-authorProfileLink?